easybuild/framework/easyblock.py

```diff
@@ -1065,7 +1065,7 @@ def make_module_req_guess(self):
             'LIBRARY_PATH': ['lib', 'lib64', 'lib32'],
             'CPATH': ['include'],
             'MANPATH': ['man', 'share/man'],
-            'PKG_CONFIG_PATH': ['lib/pkgconfig', 'share/pkgconfig'],
+            'PKG_CONFIG_PATH': ['%s/pkgconfig' % x for x in ['lib', 'lib64', 'lib32', 'share']],
```

Comment:
@hajgato: style, mostly, the order should be changed for other stuff too, indeed

for `'share/man'`, it's a single string, not a composed string, but it doesn't hurt to also use `os.path.join` there